rules/windows/process_creation/proc_creation_win_susp_microsoft_onenote_child_process.yml

```
@@ -0,0 +1,101 @@
+title: Suspicious Microsoft OneNote Child Process
+id: c27515df-97a9-4162-8a60-dc0eeb51b775
+status: experimental
+description: Detects suspicious child processes of the Microsoft OneNote application. This may indicate an attempt to execute malicious embedded objects from a .one file.
+author: Tim Rauch (rule), Elastic (idea)
+references:
+    - https://github.com/elastic/protections-artifacts/commit/746086721fd385d9f5c6647cada1788db4aea95f#diff-e34e43eb5666427602ddf488b2bf3b545bd9aae81af3e6f6c7949f9652abdf18
+date: 2022/10/21
+tags:
+    - attack.t1566
+    - attack.t1566.001
+    - attack.initial_access
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_parent:
+        ParentImage|endswith: '\onenote.exe'
+    selection_opt_name:
+        - OriginalFilename:
+            - 'rundll32.exe'
+            - 'regsvr32.exe'
+            - 'bitsadmin.exe'
+            - 'certutil.exe'
+            - 'InstallUtil.exe'
+            - 'schtasks.exe'
+            - 'WMIC.exe'
+            - 'cscript.exe'
+            - 'wscript.exe'
+            - 'cmstp.exe'
+            - 'Microsoft.Workflow.Compiler.exe'
+            - 'RegAsm.exe'
+            - 'RegSvcs.exe'
+            - 'mshta.exe'
+            - 'Msxsl.exe'
+            - 'IEExec.exe'
+            - 'cmd.exe'
+            - 'powershell.exe'
+            - 'hh.exe'
+            - 'javaw.exe'
+            - 'pcalua.exe'
+            - 'curl.exe'
+            - 'ScriptRunner.exe'
+            - 'CertOC.exe'
+            - 'WorkFolders.exe'
+            - 'odbcconf.exe'
+            - 'msiexec.exe'
+            - 'msdt.exe'
```

```suggestion
            - 'RUNDLL32.exe'
            - 'REGSVR32.exe'
            - 'bitsadmin.exe'
            - 'CertUtil.exe'
            - 'InstallUtil.exe'
            - 'schtasks.exe'
            - 'wmic.exe'
            - 'cscript.exe'
            - 'wscript.exe'
            - 'CMSTP.EXE'
            - 'Microsoft.Workflow.Compiler.exe'
            - 'RegAsm.exe'
            - 'RegSvcs.exe'
            - 'MSHTA.EXE'
            - 'Msxsl.exe'
            - 'IEExec.exe'
            - 'Cmd.Exe'
            - 'PowerShell.EXE'
            - 'HH.exe'
            - 'javaw.exe'
            - 'pcalua.exe'
            - 'curl.exe'
            - 'ScriptRunner.exe'
            - 'CertOC.exe'
            - 'WorkFolders.exe'
            - 'odbcconf.exe'
            - 'msiexec.exe'
            - 'msdt.exe'
```

Note that there is no original filename set on the binary `pcalua.exe`